### PR TITLE
fix #182321: Tempo text from 2.0.3 not imported correctly

### PR DESCRIPTION
**Pull request by lasconic**

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Travise failes, https://travis-ci.org/musescore/MuseScore/jobs/362607809#L5298-L5311
Somewhat surprisingly 2.x really does not use the `met*` variants of these notes but the `unicode*` one

File: libmscore/read206.cpp [hunk 1]
```diff
@@ -1933,6 +1935,17 @@ static void readMeasure(Measure* m, int staffIdx, XmlReader& e)
...
                   segment->add(el);
                   }
+            else if (tag == "Tempo") {
+                  Element* el = Element::name2Element(tag, score);
+                  // hack - needed because tick tags are unreliable in 1.3 scores
+                  // for symbols attached to anything but a measure
+                  el->setTrack(e.track());
+                  el->read(e);
+                  TextBase* tt = static_cast<TextBase*>(el);
+                  tt->setXmlText(tt->xmlText().replace("<sym>unicode", "<sym>met"));
```
Comment (Jojo-Schmitz) [inline libmscore/read206.cpp]: Seems this is the cause for the Travis failure, not sure whether it is the tests than need adjusting?

**Comment by lasconic** [comment]
It depends. I believe starting with 2.1 we used met*

**Comment by Jojo-Schmitz** [comment]
I just created a score with 2.2 and it used the `unicode*` one (when adding the tempo via new score wizard)

**Comment by lasconic** [comment]
Werird https://github.com/musescore/MuseScore/blob/2.2.1/mscore/file.cpp#L744

**Comment by Jojo-Schmitz** [comment]
Indeed. Let me check again... copy paste from score:  <xml><sym>metNoteQuarterUp</sym> = 100 </xml>

**Pull request merged**